### ISSUE TITLE
Multiple docker-sync.yml files not working since upgrade

### ISSUE THREAD

**Issue by marcdaffern** [EugenMayer/docker-sync#331]
Previous version: 0.2.3
Upgraded to: 0.3.6

We have two files at root level, `docker-sync.grunt.yml` and `docker-sync.watch.yml`. 

These are called like so:

```
docker-sync-daemon \
    start \
    --no-logd \
    --config ./docker-sync.watch.yml
```

This was working fine until the upgrade, but now docker-sync falls over because there is no `docker-sync.yml` file at root level (because we only have the two specific files mentioned above). 

I get the error 

`error  No docker-sync.yml configuration found in your path ( traversing up ) Did you define it for your project?`

I can get it running by simply creating a `docker-sync.yml` at root, but we need to have the two different files to handle different scenarios. It now appears to be ignoring the config location setting upon start.

**Comment by EugenMayer** [comment]
this works with 0.4.x now

**Comment by marcdaffern** [comment]
Great thanks! I've updated to 0.4.0 and it's now running correctly again. 

However, when I run docker-sync clean in the new version, I get the same error as above.

**Comment by EugenMayer** [comment]
i cannot reproduce it, so you will need to provide a small as possible example..
thats what i used https://github.com/EugenMayer/docker-sync-boilerplate/tree/master/not_default_config_name

and it worked

**Comment by marcdaffern** [comment]
Hmm, odd - I cloned your repo, and in the not_default_config_name folder I ran `docker-sync start --config ./docker-sync-special.yml` and that worked fine. I ran `docker-sync clean` and I get the error `No docker-sync.yml configuration found in your path ( traversing up ) Did you define it for your project?`

Just re-checked version and it's definitely 0.4.0.

I'm not using docker-sync-clean anywhere in my code so it's not really an issue for me, I just ran it  as it was suggested when upgrading to latest version.
